cockatrice/src/gamesmodel.cpp

```diff
@@ -314,11 +314,26 @@ void GamesProxyModel::setMaxPlayersFilter(int _maxPlayersFilterMin, int _maxPlay
     invalidateFilter();
 }
 
+int GamesProxyModel::getNumFilteredGames() const
+{
+    GamesModel *model = qobject_cast<GamesModel *>(sourceModel());
```

Comment:
This segfaults when I click "show this user's games" like #3979 but inside the sourceModel() call.

Reply:
Oh yikes, thanks for the heads up! Seems reasonable for someone to cleanly revert https://github.com/Cockatrice/Cockatrice/pull/3946 since this change will undo it anyway; @ZeldaZach would you like me to create a PR?

Reply:
Yes please

Reply:
Done, #3988. I'll double check this behavior with this PR (I think it needed to be guarded behind "showfilters")

Reply:
Confirmed this should be fixed with next commit :)